### PR TITLE
Updated error messages, fixed paging bug & additional tests (unit and ivt)

### PR DESCRIPTION
**Pull request by jadecarino**

### CODE REVIEW

**Comment by jadecarino** [comment]
@KirbyKatcher @techcobweb Hi both would be good to get your thoughts on the new tests added to the IVTs. I wanted to test that the correct GAL error message was reported for different problems with the --age param but wasn't sure how to as the error output wasn't being redirected to an output file with `tee` as I thought it would. Happy to add that in if you think it would be beneficial.

**Comment by galasa-team** [comment]
Build successful

**Comment by galasa-team** [comment]
Build successful

File: test-galasactl-ecosystem.sh [hunk 1]
```diff
@@ -373,23 +373,132 @@ function runs_get_check_raw_format_output {
...
+    cd ${BASEDIR}/temp
+
+    cmd="${BASEDIR}/bin/${binary} runs get \
+    --age 1h:0h \
+    --format raw \
+    --bootstrap ${bootstrap}"
+
+    info "Command is: $cmd"
+
+    output_file="runs-get-output.txt"
```
Comment (techcobweb) [inline test-galasactl-ecosystem.sh:391]: set -o pipefail

File: test-galasactl-ecosystem.sh [hunk 2]
```diff
@@ -484,12 +593,13 @@ runs_get_check_details_format_output  $RUN_NAME
...
+runs_get_check_raw_format_output_with_from_and_to $RUN_NAME
+runs_get_check_raw_format_output_with_just_from $RUN_NAME
 
 # Check that the age parameter throws correct errors with invalid values
-runs_get_check_summary_format_output_with_invalid_age_param $RUN_NAME
-runs_get_check_summary_format_output_with_no_runname_and_no_age_param
-runs_get_check_summary_format_output_with_badly_formed_age_param
+runs_get_check_raw_format_output_with_no_runname_and_no_age_param
+runs_get_check_raw_format_output_with_invalid_age_param
+runs_get_check_raw_format_output_with_older_to_than_from_age
```
Comment (techcobweb) [inline test-galasactl-ecosystem.sh:602]: I think all the variations in terms of syntax are already verified in the junits.

I think at the IVT level we want to test that the results match up to what we thing they should be... but in the case of this age parameter, we can't do any before-after testing really, as the results are going to be time-dependent. 

So... I think what you've done is ok, though the older-than... and invalid-age tests aren't needed, given that we cover those in unit tests ? But it probably doesn't hurt to leave them there anyway.

**Review by techcobweb — APPROVED** [review]

**Review by KirbyKatcher — APPROVED** [review]

**Pull request merged**